### DISCUSSION TITLE
showing workstation   not reachable whine running sonar scanner

### DISCUSSION THREAD

**Discussion by surendraalamuru22** [learndevopsonline/learndevopsonline#1863] [Q&A]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/6e6f8f74-1792-4beb-bbd0-31f3e8efdc7b)

workstation ip:54.173.131.111

![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/dfa3393a-6d48-4582-beb6-916ee11347c0)

**Comment by chaithanya1812** [comment]
workstation is not reachable..

**Reply by surendraalamuru22** [reply]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/5ec97ae0-9362-4952-8f5b-8dd364da9e88)

iam working on it 
now you can try

**Reply by surendraalamuru22** [reply]
i stopped and restarted the workstation

**Reply by chaithanya1812** [reply]
share the new public-ip address of workstation..

**Reply by surendraalamuru22** [reply]
34.227.228.135

**Comment by chaithanya1812** [comment]
Hi @surendraalamuru22 
##For shipping code it's works
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/517c71e9-e1db-437e-bf63-c00fddb5ce12)

# cart & other components developed with nodejs, so it requires more memory ,Try to increase the workstation Instancetype

**Reply by surendraalamuru22** [reply]
ok chaitanya

**Reply by surendraalamuru22** [reply]
chaitanya its not giving the option to change instance type
![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/92f823eb-1821-49e7-8b36-d951e3b00e86)

**Reply by chaithanya1812** [reply]
## stop the instance, so that option is enabled

**Reply by surendraalamuru22** [reply]
ya instance already in stopped state

**Reply by chaithanya1812** [reply]
For spot instances, it will not have that option,
Does your instances are spot instances?

**Reply by surendraalamuru22** [reply]
yes it is spot instance only

**Reply by chaithanya1812** [reply]
oky bro, alternatively add below option to sonar cli command, it will work 
## remember it's not right option for your problem, for learning purpose we can go with it

-Dsonar.javascript.exclusions="*.js"

**Reply by surendraalamuru22** [reply]
ok chaitanya

**Reply by surendraalamuru22** [reply]
chaitanya iam getting this error

![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/ccfd710c-9b89-4477-a59e-b7fe491fb597)

**Reply by chaithanya1812** [reply]
share the new public ip address

**Reply by surendraalamuru22** [reply]
workstation ip : 54.226.59.8

**Comment by chaithanya1812** [comment]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/4716e723-9af8-4c51-9549-bc7582583cef)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/6b38c703-e87b-44f7-be28-a29d00f8c8a0)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/2461190d-d358-40ea-b43e-c4011bf71eaa)

**Reply by surendraalamuru22** [reply]
but chaitanya its not showing any project files and code coverage

![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/6f682589-697f-4c7b-93f3-a3520bfeafc9)

**Reply by surendraalamuru22** [reply]
and ealier i used sudo then it  is worked , that was the only issue it has

**Reply by surendraalamuru22** [reply]
it did not shows any files it is scanned
![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/450f8978-a8dd-4b10-af19-0d47347e8c40)

**Reply by surendraalamuru22** [reply]
workstation ip:54.87.38.124

**Reply by surendraalamuru22** [reply]
code checkout is a failure
![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/735377c7-26aa-4dab-9387-72af4814f7dc)

**Reply by chaithanya1812** [reply]
Hi @surendraalamuru22 , I already discussed because of instance_type that your are using.

**Reply by surendraalamuru22** [reply]
so do i need to create another instance with desired requirement or do i need to run any  cli command  to  work seamlessly

**Reply by chaithanya1812** [reply]
Try with another instance_type with desired requirement

**Reply by surendraalamuru22** [reply]
ok chaitanya
